README.md

```diff
@@ -99,13 +111,20 @@ Note that in the following instructions, we assume your catkin workspace (in whi
    mkdir -p ~/catkin_ws/src
    ```
 
-1. Install mavros version 0.29.0 or above. Instructions to install it from sources can be found here: https://dev.px4.io/en/ros/mavros_installation.html. If you want to install using apt, be sure to check that the version is 0.29.0 or greater.
-
-   ```bash
-   sudo apt install ros-kinetic-mavros ros-kinetic-mavros-extras
-   ```
-
-1. Install the geographiclib dataset
+1. Install MAVROS (version 0.29.0 or above).
+   > **Note:** Instructions to install MAVROS from sources can be found [here](https://dev.px4.io/en/ros/mavros_installation.html).
+   If you want to install using apt, be sure to check that the version is 0.29.0 or greater.
```

Comment:
@mrivi How would you check? It seems to me that the version on  apt 18.04 (and presumably 16.04) are up to date - or at least they appear to work  - when is this warning needed?

Reply:
I think we can remove this working. We put it there as soon as 0.29.0 was released but that's quite old now. to check the package version I think you need to find the precompiled library in you system and in that folder there should be a package.xml file containing the versioning information

Reply:
Agreed. I'll keep the version required, but not the suggestion that you need to check it.